packages/addons/service/rsyslog/source/settings-default.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?> <settings>
+    <category label="1000">
+		<setting label="30010" type="lsep"/>
+		<setting type="sep" />
+		<setting id="RSYSLOG_IP_NAME_FQDN" type="text" label="30011" values="" default="" />
```

Comment:
label, type, id would be cleaner
text, number and bool have no values attribute
http://kodi.wiki/view/Add-on_settings